app/Console/Commands/Setup/SetupSyncPteroCommand.php

```diff
@@ -59,6 +60,16 @@ public function handle()
         Egg::syncEggs();
         $bar->advance();
 
+        $this->newLine();
+        $this->line('Syncing Servers..this might take a while.');
```

Comment:
This command doesn't do anything if there are no servers, this command is (currently) used only used during installation. this is a bit confusing

Reply:
this sync does not make new servers right, cuss when migrating you have no users
so you are missing the required user_id's